README.md

```diff
@@ -23,20 +23,20 @@ npm install changelog-parser
 This module exports a single function. It supports both callbacks and promises.
 
 ```js
-var parseChangelog = require('changelog-parser')
+const parseChangelog = require("changelog-parser");
```

Comment:
please don't change all of the formatting in the readme to suit your personal style preferences 🙏

Reply:
my bad, formatOnSave did that.

Reply:
np, thanks for fixing!